security/ipsw/Portfile

```
@@ -3,30 +3,36 @@
 PortSystem              1.0
 PortGroup               golang 1.0
 
-go.setup                github.com/blacktop/ipsw 3.1.461 v
+go.setup                github.com/blacktop/ipsw 3.1.462 v
 github.tarball_from     archive
 revision                0
 categories              security devel
 license                 MIT
-platforms               {darwin >= 17}
+platforms               {darwin >= 19}
 installs_libs           no
 maintainers             {@TheRealKeto gmail.com:therealketo} openmaintainer
 
 description             iOS/macOS Research Swiss Army Knife
 long_description        {*}${description}. Everything you need to start \
                         researching Apple security and internals.
 
-checksums               rmd160  147979bad0e97e464cf7c3b4ffaea9210cd51b6c \
-                        sha256  07046f136abc08c7e10c9645f78bc0e1895ede941660fa6b04c258972a5c9530 \
-                        size    4048865
+checksums               rmd160  c76625b209f70f05876664d5153de3727c93168b \
+                        sha256  b73c9e029e1e25e9a7c5cff018271b9e693e27e5beab29c290c7ad5aaae4f00e \
+                        size    4051704
 
 patch.pre_args          -p1
 patchfiles              reproducible-build.diff
 
+depends_build-append    port:pkgconfig
```

pkgconf is an alternate implementation of the `pkg-config` program; please allow that port to satisfy this dependency:

```suggestion
depends_build-append    path:bin/pkg-config:pkgconfig
```